cmd/root.go

```diff
@@ -123,6 +123,13 @@ func init() {
 	viper.BindPFlag("skip-workspace-crud",
 		rootCmd.PersistentFlags().Lookup("skip-workspace-crud"))
 
+	// Support for Session Cookie
+	rootCmd.PersistentFlags().String("kong-session-cookie-path", "",
+		"Absolute path for the session-cookie file for auth with Admin Server.\n"+
+			"You may also need to pass in as header the User-Agent that was used to create the session.")
```

Comment:
Update this text as well.
Please mention the format of the cookie file path (netscape cookie format I think).

Question: what happens if the user-agent is not specified?

Reply:
It gives a 401. This took me a while to figure out. Looks like Kong associates sessions and user-agent. My guess is that when Kong sees a different user agent, it assumes that the session was hijacked and sends a 401

Reply:
Yikes. That may introduce additional friction that we can't solve for. Let's ship this and come back here if this PR doesn't achieve the desired outcome for our customers.